cddl/conditional-endorsement-series-triple-record.cddl

```diff
@@ -1,4 +1,8 @@
 conditional-endorsement-series-triple-record = [
-  condition: stateful-environment-record
+  condition: [
+    environment: environment-map,
+    ? claims-list: [ + measurement-map ],
+    ? authorized-by: [ + $crypto-key-type-choice ]
```

Comment:
This changes the parsing complexity to at least LL2, and is potentially ambiguous due to the extensibility of $crypto-key-type-choice. Optional elements of a heterogeneous array type are an anti-pattern. If you want claims-list to be optional as well as authorized-by, the way to do it is to make the group of the two optional and allow authorized-by to have an empty list.